anthos-bm-gcp-terraform/modules/init/main.tf

```diff
@@ -102,9 +102,15 @@ resource "null_resource" "exec_init_script" {
     destination = "${local.home_dir}/login.sh"
   }
 
+  provisioner "file" {
+    source      = var.nfs_yaml_path
+    destination = "${local.home_dir}/.nfs-csi.yaml"
```

Comment:
Why are we making a `.` hidden file? Can we make it just a regular file?

Reply:
@Shabirmean when you have time, please take a look as this is the last open item.  Thanks!

Reply:
I agree it will be an overkill to pass the `nfs` variable to the init module. I think it is okay to keep the file `unhidden`. We can add a comment at the top of the template file stating what it is.